libs/test-common/src/main/java/org/codice/ddf/test/common/configurators/KarafOptions.java

```diff
@@ -90,6 +94,9 @@ private Option getVmOptions() {
 
   private Option getPaxExamOptions() {
     return composite(
+        mavenBundle(
+            "org.apache.servicemix.bundles", "org.apache.servicemix.bundles.hamcrest", "1.3_1"),
```

Comment:
✏️  lol and literally found a perfect example of what I was talking about in my above nit ✏️  regarding symbolic names